src/Microsoft.Sbom.Api/Executors/GenerationResult.cs

```diff
@@ -18,12 +17,17 @@ public class GenerationResult
 
     public Dictionary<IManifestToolJsonSerializer, List<JsonDocument>> SerializerToJsonDocuments { get; set; }
 
-    public IEnumerable<ISourcesProvider> SourcesProviders { get; set; }
+    public bool JsonArrayStarted { get; set; }
```

Comment:
If we're only setting this in the constructor, we don't need the setter
```suggestion
    public bool JsonArrayStarted { get; }
```